release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
+* You can filter Helm charts based on their certification level and see all the Helm charts in the *Developer Catalog*.
```

Comment:
I think bullet lists ought to be scannable and quickly understood. I think a good way to make a bullet lists scannable is to construct each sentence for the bullet point in the same way. 

I think instead of mostly saying `You can...`, we can simply lead with the action. Action oriented makes the bullet points more scannable. See my rewrites and let me know what you think :)